pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -316,6 +316,81 @@ self: super: {
     '';
   });
 
+  coq_nvim = super.coq_nvim.overrideAttrs (old: {
+    passthru.python3Dependencies = let
+
+      # Extract the version of the dependencies from the requirements.txt file
+      requirements = builtins.readFile "${old.src}/requirements.txt";
+      requirementsLines = lib.splitString "\n" requirements;
+
+      extractLine = pname: lib.findSingle
+        (lib.strings.hasPrefix pname)
+        "none"
+        "multiple"
+        requirementsLines;
+
+      extractDepVersion = pname:
+        let
+          line = extractLine pname;
+          hashPart = lib.lists.last (lib.splitString "/" line);
+          hash = lib.strings.removeSuffix ".tar.gz" hashPart;
+        in hash;
+    in ps: with ps; [
+      pynvim
+      pyyaml
+      (let
+        version = extractDepVersion "pynvim_pp";
+      in buildPythonPackage {
+        pname = "pynvim_pp";
+        inherit version;
+        format = "pyproject";
+        propagatedBuildInputs = [ setuptools pynvim ];
+        src = fetchFromGitHub {
+          owner = "ms-jpq";
+          repo = "pynvim_pp";
+          rev = version;
+          fetchSubmodules = false;
+
+          # This maps the current version of pynvim_pp to the sha256 of its source.
+          # Hence, if the dependency version changes in the requirements.txt, the build will fail.
+          hash = { "456765f6dc8cef6df39ae3e61c79585258d38517" = "sha256-edDDzxR60ILFbv1CnYGxW/sJDMeRIFPjmTeiMRdvA3k="; }.${version};
+        };
+        meta = with lib; {
+          homepage = "https://github.com/ms-jpq/pynvim_pp";
+          license = licenses.gpl3;
```

Comment:
Either use `licenses.gpl3Only` (if this is the only supported version) or `licenses.gpl3Plus` (if any version 3+ is supported).

Reply:
I am not sure which one is the case for those two repos...